static/src/stylesheets/module/commercial/_adverts-capi.scss

```diff
@@ -70,10 +72,49 @@
     }
 }
 
+.adverts--tone-supported {
+    background: none;
+
+    &::before {
+        content: '';
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        border-top: 1px solid $news-main-2;
+
+        @include mq(tablet) {
+            left: calc(50% - 370px);
```

Comment:
I'm assuming the idea is to horizontally center something with a 50% width. Could something like the following handle all viewport sizes?

````
left: 0;
width: 50%;
transform: translateX(50%);
````